releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,21 @@
 
 [Image][image] | [Sample][sample]
 
+## 2.2.0 - MM DD, 2025
+
+### Enhancements
+
+* Added support for on-demand refresh, which enables you to trigger an on-demand refresh to get the latest data from App Configuration and Key Vault by modifying `metadata.annotations` section. This enhancement makes the refresh process more flexible and responsive to real-time needs. [#87](https://github.com/Azure/AppConfiguration-KubernetesProvider/pull/87)
+* Added support for the `variants`, `allocation`, and `telemetry` properties for feature flags added by the [Microsoft.FeatureManagement.*](https://github.com/Azure/AppConfiguration?tab=readme-ov-file#feature-management-libraries) libraries. While variant feature flags can still be toggled on or off, they also allow for different configurations, ranging from simple primitives to complex JSON objects. Variant feature flags are particularly useful for feature rollouts, configuration rollouts, and feature experimentation (also known as A/B testing).

```

Comment:
> Added support for the `variants`, `allocation`, and `telemetry` properties for feature flags added by the [Microsoft.FeatureManagement.*](https://github.com/Azure/AppConfiguration?tab=readme-ov-file#feature-management-libraries) libraries.

That's a lot of details. Technically, feature flags are not added by FM libraries. How about simplifying it to "Added support for variant feature flags."?